examples/1_compose_prove_verify.rs

```diff
@@ -270,7 +270,12 @@ fn main() {
         .unwrap();
 
     // Now we can finally preprocess the circuit that we've built.
+<<<<<<< HEAD
```

Comment:
Please fix this

Reply:
You left the review just when I merged your PR. That's why this is there.